feedbag/feedbag.go

```diff
@@ -34,7 +35,9 @@ func Start(port, templatesDir string) error {
 
 	// Process our templates
 	TemplatesDir = templatesDir
-	Templates = getTemplates(TemplatesDir)
+	var err error
+	Templates, err = template.ParseDir(TemplatesDir)
```

Comment:
if I say `Templates, err := ` then it will define a local variable of Templates instead of overwriting the global variable we defined earlier